Wissance.MossbauerLab.Watcher/Wissance.MossbauerLab.Watcher.Services/Notification/TelegramNotifier.cs

```diff
@@ -1,11 +1,49 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Threading.Tasks;
 
+using Telegram.Bot;
+using Telegram.Bot.Polling;
+using Telegram.Bot.Requests;
+using Telegram.Bot.Types;
+
+using Wissance.MossbauerLab.Watcher.Common.Data;
+using Wissance.MossbauerLab.Watcher.Services.Config;
+
 namespace Wissance.MossbauerLab.Watcher.Services.Notification
 {
-    public class TelegramNotifier
+    public class TelegramNotifier : ISpectrumReadyNotifier
     {
+        private readonly ApplicationConfig _config;
+        private const string SpectrumAutoSaveMailTemplate = @"Templates/autosaveNotifications.html";
+
+        public TelegramNotifier(ApplicationConfig config)
+        {
+            _config = config;
+        }
+        public async Task<bool> NotifySpectrumSavedAsync(IList<SpectrumReadyData> spectra)
+        {
+            ITelegramBotClient client = new TelegramBotClient("6253527316:AAHrOysU6QKubqCg51aHuVCNBf3PtD3nGBU");
+           
+            var targetGroupName = _config.NotificationSettings.TelegramSettings.NotificationGroupName;
```

Comment:
We don't use var's either because we always know our types and this helps to make code more readable

Reply:
Ok, add editor config with codestyle preferences

Reply:
You can use type hints. In my IDE code looks like this:
![image](https://user-images.githubusercontent.com/2220280/218772417-99db7dc0-5663-4d26-a0f3-50fef44df3d5.png)